thoth/storages/graph/models_base.py

```diff
@@ -73,7 +73,14 @@ def to_dict(self, without_id: bool = True) -> dict:
             if without_id and column.name == "id":
                 continue
 
-            result[column.name] = str(getattr(self, column.name)) if getattr(self, column.name) else None
+            if not getattr(self, column.name):
+                result[column.name] = None
+            else:
+                value = getattr(self, column.name)
+                if isinstance(value, (datetime.datetime, datetime.date)):
+                    result[column.name] = str(value)
+                else:
+                    result[column.name] = value
```

Comment:
I still cannot sync and I understood the reason. It should be:
```
            value = getattr(self, column.name)

            if isinstance(getattr(self, column.name), bool):
                result[column.name] = value

            elif not getattr(self, column.name):
                result[column.name] = None

            else:

                if isinstance(value, (datetime.datetime, datetime.date)):
                    result[column.name] = str(value)
                else:
                    result[column.name] = value
```

because `not getattr(self, column.name)` consider also values with False, and in that case we do not want to assign None but False. (For this reason we receive error `sqlalchemy.exc.IntegrityError: (psycopg2.errors.NotNullViolation) null value in column "debug" violates not-null constraint`, it needs to be bool, not None)

Reply:
Thanks, resolved in 4a47db4